docs/hugo/content/reference/breaking-changes.md

```diff
@@ -0,0 +1,423 @@
+# Breaking Changes in beta.4
+
+In the 'beta.4' release of Azure Service Operator (ASO) we are pivoting to using the Azure Swagger API Specifications as our sole source of truth for our code generator.
+
+This change brings with in a significant improvement in fidelity - the code we generate is now much closer to what the Azure Swagger API Specifications describe.
+
+Unfortunately, this change brings with it a few breaking changes, listed in this document. 
+
+We expect that most users will find their resources are unaffected by these changes. However, if you are using a resource that is affected, you will need to take action to migrate your resources to the new format.
+
+The impact of these breaking changes falls into two categories, *immediate migration required* and *upgrade when modified*, distinguished by when you need to take remedial action.
+
+### Immediate migration required
+
+As a part of upgrading ASO in your cluster, you will need to modify these resources as described. In all cases, the changes are straightforward and can be done with a simple text editor.
+
+The process to follow is as follows:
+
+1. Annotate the resource with `serviceoperator.azure.com/reconcile-policy: skip` to prevent ASO from trying to reconcile the resource while you are upgrading.
+2. Download the current YAML for the resource using `kubectl` if you don't have it elsewhere.
+3. Delete the resource from your cluster using `kubectl delete`. Your Azure resource will be left untouched because of the `reconcile-policy` annotation you added above.
+4. Upgrade the version of ASO in your cluster.
```

Comment:
add the link to helm upgrade here? or steps to follow for upgrading?

Reply:
Added link to our docs page on upgrades.